pkg/migrations/sql/idempotent/base.sql

```diff
@@ -250,7 +253,8 @@ BEGIN
         ) WITH (autovacuum_vacuum_threshold = 100, autovacuum_analyze_threshold = 100)
     $$, NEW.table_name, label_id, NEW.id);
    EXECUTE format('GRANT SELECT ON TABLE SCHEMA_DATA_SERIES.%I TO prom_reader', NEW.table_name);
-   EXECUTE format('GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE SCHEMA_DATA_SERIES.%I TO prom_writer', NEW.table_name);
+   EXECUTE format('GRANT SELECT, INSERT ON TABLE SCHEMA_DATA_SERIES.%I TO prom_writer', NEW.table_name);
```

Comment:
The set of perms is repeated all over the place...perhaps you could just make a function like `grant_promscale_default_perms(schema, table)`?

Reply:
only used in two places in same function. Don't think it's worth refactoring

Reply:
fair enough.